master/static/srv/set_user_display_name.sql

```diff
@@ -0,0 +1,12 @@
+WITH confusable_users AS (
+  SELECT COUNT(*) AS count
+  FROM users
+  WHERE $2 != ''
```

Comment:
question: could this `$2 != ''` conditional cancel out the `LOWER(username)` conditional?

Reply:
This is to find username conflicts -- if the set username is blank it should be 0, otherwise we look at the other WHERE clauses